packages/maskbook/src/plugins/PluginUI.ts

```diff
@@ -15,12 +15,14 @@ import { TraderPluginDefine } from './Trader/define'
 import { Election2020PluginDefine } from './Election2020/define'
 import { TransakPluginDefine } from './Transak/define'
 import { COTM_PluginDefine } from './COTM/define'
+import { NFTPluginsDefine } from './NFT/define'
 import { ITO_PluginDefine } from './ITO/define'
 
 plugins.add(WalletPluginDefine)
 plugins.add(GitcoinPluginDefine)
 plugins.add(RedPacketPluginDefine)
 plugins.add(FileServicePluginDefine)
+plugins.add(NFTPluginsDefine)
```

Comment:
should it be enabled by default? @guanbinrui have idea?

Reply:
enable by default - yep on my side

Reply:
I'd love to have a flag to be added like other plugins.